js/viz/docs/docchartseries.js

```diff
@@ -1229,8 +1227,7 @@ var barSeries = {
         /**
         * @name dxChartSeriesTypes_barseries_aggregation_method
         * @publicName method
-        * @type string
-        * @acceptValues 'avg'|'count'|'max'|'min'|'sum'|'custom'
+        * @type ChartSimpleSeriesAggregationMethod
```

Comment:
I don't see definition of ChartSimpleSeriesAggregationMethod type.

Reply:
I've added it

Reply:
"Enums.ChartSimpleSeriesAggregationMethod"